assets/js/modules/sign-in-with-google/components/settings/SettingsEdit.stories.js

```diff
@@ -44,12 +47,21 @@ function Template() {
 
 export const Default = Template.bind( null );
 Default.storyName = 'Default';
-Default.scenario = {
-	label: 'Modules/SignInWithGoogle/Settings/SettingsForm/Default',
+Default.scenario = {};
+
+export const NewUserAccountsEnabled = Template.bind( null );
+NewUserAccountsEnabled.storyName = 'New Accounts Enabled';
+NewUserAccountsEnabled.args = {
+	anyoneCanRegister: true,
 };
+NewUserAccountsEnabled.scenario = {};
```

Comment:
This story/VRT is visually exactly the same as one where the banner was dismissed. This isn't a visual test but a unit test; we don't need a separate VRT for this one.